libraries/anaconda3-buildtime/Dockerfile

```diff
@@ -1 +1,3 @@
 COPY --chown=0:0 anaconda3-buildtime/context/build /usr/local/bin/algorithmia-build
+RUN conda create --prefix ANACONDA_ENV && \
```

Comment:
So I'm confused here. `ANACONDA_ENV` isn't defined here, right?
Secondly, can we just become `USER algo` before this `RUN` operation and then we won't need to do the chown as part of the RUN command

Reply:
Yeah, to follow up on that second point I've unresolved the initial thread: https://github.com/algorithmiaio/langpacks/pull/101#discussion_r312192350 and would appreciate clarification -- if only for my own understanding -- on why it would not be possible to use `USER algo`, if that is the case.

Reply:
environment variables set in dockerfiles propegate automatically into bash scripts,new commit will rectify this.

Reply:
And the $ANACONDA_ENV should be a variable here as well. And wondering why we can't run this as user algo still

Reply:
think this should be using the env variable instead of the literal

Reply:
user algo would disrupt this variable, I'd have to redeclare it after I switch to user algo. This is a variable, it's defined in the anaconda35 base image, if you guys would like I can redefine it at the top of both the runtime and the buildtime dockerfiles for legibility, but it would technically be redundant. Let me know what you think!